tools/restore-backups-omaha.sh

```diff
@@ -88,14 +88,17 @@ fillVolumeInfo() {
       #fi
       local pvc="postgres-pvc-v12"
       local pv="pvc-a87f0c51-3eb8-48d6-94ae-a086bc4f9f79"
-      local tagspec="{Key=kubernetes.io/cluster/DW,Value=owned},{Key=kubernetes.io/created-for/pv/name,Value='pvc-a87f0c51-3eb8-48d6-94ae-a086bc4f9f79'},{Key=kubernetes.io/created-for/pvc/namespace,Value=omaha},{Key=kubernetes.io/created-for/pvc/name,Value='postgres-pvc-v12'}}"
+      local tagspec="{Key=kubernetes.io/cluster/DW,Value=owned}"
+            tagspec="$tagspec,{Key=kubernetes.io/created-for/pv/name,Value=$pv}"
+            tagspec="$tagspec,{Key=kubernetes.io/created-for/pvc/name,Value=$pvc}"
+            tagspec="$tagspec,{Key=kubernetes.io/created-for/pvc/namespace,Value=omaha}"
     elif ! [ -z "$data" ] ; then
       local az=$data
     fi
   done
   unset IFS
 
-  if [ -z "pvc-a87f0c51-3eb8-48d6-94ae-a086bc4f9f79"] || [ -z "${pvc2Pod['pvc-a87f0c51-3eb8-48d6-94ae-a086bc4f9f79']}" ] || [ -z "$az" ] ; then
+  if [ -z "${pvc2Pod['pvc-a87f0c51-3eb8-48d6-94ae-a086bc4f9f79']}" ] || [ -z "$az" ] ; then
```

Comment:
This seems strange: the variable is called pvc2pod, yet we give a pv here... 
Shouldn't we rather call it with [ -z "${pvc2Pod[$pvc]}" ] (like for Sentry)